src/views/StoryViewer.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="viewer" :style="currentFrame.style">
+    <p v-if="currentFrame.text">{{ currentFrame.text }}</p>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex'
+export default {
+  name: 'StoryViewer',
+  computed: mapGetters(['currentFrame']),
+  methods: {
+    ...mapActions(['selectFrame']),
+    playNextFrame: (currentFrameId, nextFrame) => {
+      const timeoutId = setTimeout( () => {
+        nextFrame(currentFrameId + 1)
+        clearTimeout(timeoutId)
+      }, 2000)
+    }
+  },
+  mounted: function() {
+    const currentFrameId = this.$store.getters.currentFrame.frameId
```

Comment:
Since you have mapGetters(['currentFrame']) above you should be able to write this.currentFrame.frameId